### ISSUE TITLE
Add Matrix & Riot

### ISSUE THREAD

**Issue by Biep** [libreprojects/libreprojects#196]
Libre federated privacy-protecting social interaction protocol and system.
http://matrix.org
http://riot.im

**Comment by jancborchardt** [comment]
Yep, Riot.im should be added (Matrix not, one is enough). :) We can just replace the Freenode IRC webchat with it, because it’s much more usable. Do you want to open a pull request?

**Comment by Biep** [comment]
If I knew what that was and how to do it..

**Comment by jancborchardt** [comment]
@Biep the process is described at "Adding new projects" here :) https://github.com/libreprojects/libreprojects#adding-new-projects

Just find the line of "Freenode IRC" in `js/data.json` and overwrite it with Riot info. Also add the logo of Riot of course. Let me know if that works or if you need help. :)
